scripts/gha/dismiss_reviews.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A utility to dismiss PR reviews.
+
+USAGE:
+  python scripts/gha/it_workflow.py --stage <stage> \
```

Comment:
`python scripts/gha/dismiss_reviews.py`

Reply:
Eep :)

Reply:
BTW the github.py API is really nice. I'm going to move the other scripts I wrote over to use it soon.